lib/matplotlib/font_manager.py

```diff
@@ -832,10 +821,7 @@ def set_family(self, family):
         """
         if family is None:
             family = rcParams['font.family']
-        if is_string_like(family):
-            family = [six.text_type(family)]
-        elif (not is_string_like(family) and isinstance(family, Iterable)):
-            family = [six.text_type(f) for f in family]
+        family = _normalize_font_family(family)
```

Comment:
Minor thing, but this could be just `self._family = _normalize_font_family(family)`